package.json

```diff
@@ -19,7 +19,7 @@
     "loader-utils": "^1.1.0",
     "magic-string": "^0.19.1",
     "source-map": "^0.5.6",
-    "typescript": "^2.3.2"
+    "typescript": "^2.4.0-dev.20170608"
```

Comment:
why do we need this version? rolling this out while it depends on a nightly build of typescript would be very problematic.